lib/mirrors.dart

```diff
@@ -177,6 +177,16 @@ class ReflectiveTypeInspectorService implements TypeInspectorService {
         continue;
       }
 
+      var declaration = new _MirrorDeclaration(cls, member);
+
+      // TODO: Change to the following to when fixing to ignore cases where
+      // static methods have the same name
+      // https://github.com/dart-lang/smoke/issues/23
```

Comment:
Sorry, I don't have the full context (I'm currently WFH and unfortunately I wasn't able to find Jake's original comments), but from the description in #23, I believe this is OK, so we could be OK without this TODO.